### DISCUSSION TITLE
Help please! How to write Rotor Resistance and Rotor Offset for Renault Zoe.

### DISCUSSION THREAD

**Discussion by pedroeltoro** [cedricp/ddt4all#959] [Q&A]
Hi - I've replaced a defective inverter (Command Elec V) in a 5AQ Renault Zoe 5AQ and now I need to code in the factory recorded values. On the engine mount I have Rr= 0B3163 and 0a=01CC07. 
I can connect to the vehicle perfectly with my OBD cable, but I simply struggle to enter the above data in the appropriate boxes in DDT4all to get the errors to clear. Are there any checksums to enter, or do I need to convert hex to binary, etc? 

Is anyone able to guide a newb through this please? Thanks

**Comment by matzel82** [comment]
Original....I put the signs and numbers in. That's it.

Do you have enter the expert-mode ?

**Reply by pedroeltoro** [reply]
I'm in expert mode, yes.

**Comment by pedroeltoro** [comment]
I'm in Expert Mode, I think.  For $703 I need to put something in the second field, but not sure what should go in there. The sticker from the motor case is also attached in case I'm missing something.
![IMG_0864](https://github.com/cedricp/ddt4all/assets/4151646/c2fccdbf-8f43-41e9-ab6e-a4f40a321aed)
![IMG_0871](https://github.com/cedricp/ddt4all/assets/4151646/aac7da68-c907-46bb-898b-688b7c05b8f4)

**Comment by AndyIreland** [comment]
can you just put the values in that was read, into the write boxes?
![image](https://github.com/cedricp/ddt4all/assets/49172217/6efa0199-d518-40f1-97d3-0a6a661ad0e7)

**Reply by pedroeltoro** [reply]
Thank you for your help so far. $7050 is not accepting any data. I can write in $703B, and have written this to the ECU, but I cannot clear the DTC codes:

 - Sending request :DataWrite.($7037) Rotor position offset learning + CRC written by diag tool
Delay 0 ms
Sending ELM request:2E 70 37 E6 03 00
ELM response : 6E 70 37 
Sending ELM request:1902AF
ELM response : 59 02 FF 08 47 98 28 08 4A 41 2B 08 4B 41 2B 
Clearing DTC information
Sending ELM request:14FFFFFF
Bad ELM response : WRONG RESPONSE : NR: Conditions Not Correct Or Request Sequence Error(7F1422)
ELM response : WRONG RESPONSE : NR: Conditions Not Correct Or Request Sequence Error(7F1422)
Clear DTC failed

**Comment by AndyIreland** [comment]
try setting CAN Timeout to 600ms then refresh then try writing into box again and use 'einstein' and try writing then.

![image](https://github.com/cedricp/ddt4all/assets/49172217/9151413b-e0bb-4484-83e5-cc56976485df)

You may need to also have the car in a certain mode.
Sometimes its with 'engine' off 
Sometimes its 'keycard in slot and long press on start' but not actually start the engine
sometimes its gear shifter to 'D' without engine started
Sometimes its N without engine started

**Comment by KarelSvo** [comment]
I recommend using Pyren.  
![qf454fd](https://github.com/cedricp/ddt4all/assets/30380693/5faed945-df9e-435b-b897-cb1281aeee63)
You don't have to enter a CRC there.

**Reply by AndyIreland** [reply]
seconded - if something cannot be changed in DDT4ALL sometimes its good to try PyREN
